app/views/layouts/blacklight/base.html.erb

```diff
@@ -19,7 +19,7 @@
   <body class="<%= render_body_class %>">
   <%= render partial: 'shared/header_navbar' %>
 
-  <div id="main-container" class="<%= container_classes %>">
+  <div id="main-container" class="<%= container_classes %>" role="main" aria-label="<%= t('blacklight.main.aria.main_container') %>">
```

Comment:
Did you mean to make the `div` a `main` tag too, as per https://github.com/projectblacklight/arclight/issues/838?

Reply:
Ah, yes. Thanks for catching that.